### PR TITLE
Update various spacings

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

There's some buggy behaviour where our webapp inherits the `line-height` from the `lego-editor` of `1.8`. I've tried to update most places that are affected by this if we ever update `lego-editor` with the latest `lego-bricks` variables ..

# Result

Most changes are barely noticeable.

- [x] Changes look good on both light and dark theme.
- [x] Changes look good with different viewports (mobile, tablet, etc.).
- [x] Changes look good with slower Internet connections.

<table>
    <tr>
        <td>Before</td>
        <td>After</td>
    </tr>
    <tr>
        <td>
            image
        </td>
        <td>
			image
        </td>
    </tr>
</table>

# Testing

- [x] I have thoroughly tested my changes.

All changes were tested. That's why there's not that many ...

### CODE REVIEW

**Comment by norbye** [comment]
Except for the random unrelated fixes (that I love<3) this feels like a sort of hacky solution..

Don't have my computer handy so I can't check rn, but have you tried to investigate why these variables are inherited? I tried to browse the lego-editor repo and most of the styling seemed to be scoped (except for the styling that is imported from lego-webapp and thus might be outdated at some point - we might want to look into a way to import the variables without any of the css rules)

**Comment by norbye** [comment]
> > this feels like a sort of hacky solution..
> 
> 
> 
> Are you referring to the places where I've updated the `line-height` to the "correct" `1.3`? I know we should fix the cascading bug from the editor package, but when we do, these places will break. There's already a lot of `line-height: 1.3` around in our webapp because of this, and it won't be difficult to just remove all of them when we've resolved the bug.

Ahh, then I might've judged that one a bit prematurely.. sounds good

**Review by norbye — APPROVED** [review]

File: app/routes/forum/components/ForumEditor.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ const ForumEditor = () => {
         {({ handleSubmit }) => (
           <Form onSubmit={handleSubmit}>
             <Field
-              placeholder="Title"
+              placeholder="Tips til utveksling, hvordan velge veileder eller er indøk virkelig homo?"
```
Comment (falbru) [inline app/routes/forum/components/ForumEditor.tsx:87]: 🤔

**Review by falbru — APPROVED** [review]
Looks good

File: app/routes/forum/components/ForumEditor.tsx (hunk 1 above)
Comment (Bestem0r) [reply app/routes/forum/components/ForumEditor.tsx:87]: Yeah, maybe rephrase this?

**Review by Bestem0r — APPROVED** [review]
Looks good to me

**Review by itsisak — APPROVED** [review]

File: app/routes/forum/components/ForumEditor.tsx (hunk 1 above)
Comment (ivarnakken) [reply app/routes/forum/components/ForumEditor.tsx:87]: oh yea that wasn't _really_ supposed to get added 😆
